src/lwc/geFormField/geFormField.js

```diff
@@ -71,6 +71,14 @@ export default class GeFormField extends LightningElement {
         }
     }
 
+    @wire(getObjectInfo, { objectApiName: DATA_IMPORT })
+    wiredDataImportInfo(response) {
+        if(response.data && this.isTrueFalsePicklist) {
+            this.dataImportDescribeInfo = response.data;
+            this._recordTypeId = this.recordTypeId();
+        }
+    }
```

Comment:
There was some discussion around using the already retrieved Data Import object info object in geFormRenderer to append the element that gets passed down so as not to need to get the DI object info here in geFormField.  Did that not work out?

Reply:
I think you're right, we could pass down the Data Import default recordTypeId on the element (that's what we need here), but that does muddy the element's data a bit.